src/lib/util/Type.js

```diff
@@ -1,5 +1,5 @@
-const util = require('./util');
-const { getPromiseDetails } = process.binding('util');
+require('v8').setFlagsFromString('--allow-natives-syntax');
+const getPromiseDetails = require('vm').runInThisContext('p => p instanceof Promise ? { status: %PromiseStatus(p), result: %PromiseResult(p) } : null');
```

Comment:
The code misses a semicolon:
```suggestion
const getPromiseDetails = require('vm').runInThisContext(
	'p => p instanceof Promise ? { status: %PromiseStatus(p), result: %PromiseResult(p) } : null;');
```